tests/auto/perfdata/tst_perfdata.cpp

```diff
@@ -212,6 +212,7 @@ void TestPerfData::testFiles_data()
     const auto files = {
         "vector_static_clang/perf.data",
         "vector_static_gcc/perf.data",
+        "vector_static_gcc/perf.lbr.data",
```

Comment:
please keep the LBR stuff out of this merge request - it's not directly relevant for the disassembler. please put that together with any other fixes to the LBR support into a separate merge request. and don't forget to actually include the expected test output too - right now this test will always fail as its missing the expected data